core/modules/saver-handler.js

```diff
@@ -172,18 +179,91 @@ SaverHandler.prototype.saveWiki = function(options) {
 				$tw.notifier.display(self.titleSavedNotification);
 				if(options.callback) {
 					options.callback();
-				}
-			}
-		};
-	// Call the highest priority saver that supports this method
-	for(var t=this.savers.length-1; t>=0; t--) {
-		var saver = this.savers[t];
-		if(saver.info.capabilities.indexOf(method) !== -1 && saver.save(text,method,callback,{variables: {filename: variables.filename}})) {
-			this.logger.log("Saving wiki with method",method,"through saver",saver.info.name);
-			return true;
-		}
-	}
-	return false;
+        }
+      }
+    };
+  // Process preferred if any
+  var ignorePreferred = null
+  var preferredSaver = $tw.wiki.getTiddler("$:/config/PreferredSaver")
+  if (preferredSaver !== null && preferredSaver !== undefined) {
+    var title = preferredSaver.getFieldString("text")
```

Comment:
We use 4-character tabs not spaces to indent code

Reply:
General Comment below